src/shims/dir.props

```diff
@@ -17,13 +17,16 @@
     <NetFxReference Include="System.ComponentModel.Composition" />
     <NetFxReference Include="System.ComponentModel.DataAnnotations">
       <StrongNameSig>MSSharedLib</StrongNameSig>
-    </NetFxReference> 
+    </NetFxReference>
     <NetFxReference Include="System.Data" />
     <NetFxReference Include="System.Drawing" />
     <NetFxReference Include="System.IO.Compression.FileSystem" />
     <NetFxReference Include="System.Net" />
     <NetFxReference Include="System.Numerics" />
     <NetFxReference Include="System.Runtime.Serialization" />
+    <NetFxReference Include="System.ServiceModel.Web">
```

Comment:
System.ServiceModel.Web name is miss-leading as it is just the JsonDataContractSerializer, that name was what it shipped as in SL.